wasm/lib/Cargo.toml

```diff
@@ -10,6 +10,10 @@ edition = "2018"
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+[features]
+default = ["freeze-stdlib"]
+freeze-stdlib = ["rustpython-vm/freeze-stdlib"]
```

Comment:
Nice usage of options! I think the name is good. Another viable name for this option might be `include-stdlib` to indicate the inclusion of the the standard library for people who are unfamiliar with the concept of freezing. But I think most people will probably know the idea of freeze.

This looks good to me!